### PR TITLE
New app instance

### PR DESCRIPTION
**Pull request by fabriziovigevani**
Points to the new conviction voting app instance.

We need to redeploy the subgraph before re-deploying frontend.

Also fetches HNY price from Uniswap's v2 subgraph

### CODE REVIEW

File: src/components/AppLoader.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import { SyncIndicator } from '@1hive/1hive-ui'
+
+export default function AppLoader() {
+  return <SyncIndicator visible label="Loading app" />
```
Comment (rperez89) [inline src/components/AppLoader.js:5]: do you think we need this file just for this?

File: src/hooks/useUniswapHNYPrice.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  useEffect(() => {
+    let cancelled = false
+    let retryTimer
+    async function fetchPrice() {
+      try {
+        const result = await graphqlClient.query(HNY_PRICE_QUERY).toPromise()
+
+        const { pair } = result?.data
+        const hnyPrice = pair.token1Price
```
Comment (rperez89) [inline src/hooks/useUniswapHNYPrice.js:32]: should we do`pair?.token1Price`?

**Review by rperez89 — APPROVED** [review]
LGTM!

File: src/components/AppLoader.js (hunk 1 above)
Comment (fabriziovigevani) [reply src/components/AppLoader.js:5]: Yes, this is just a placeholder for now. But Fiore is working on a custom loader so i though it was a good idea to already separate the component

File: src/hooks/useUniswapHNYPrice.js (hunk 2 above)
Comment (fabriziovigevani) [reply src/hooks/useUniswapHNYPrice.js:32]: Ah actually we could return if `!result?.data`. If there;s data we can make sure that `token1Price` is available.

File: src/components/AppLoader.js (hunk 1 above)
Comment (rperez89) [reply src/components/AppLoader.js:5]: cooool

**Pull request merged**